dashboard/app/models/lesson.rb

```diff
@@ -707,6 +709,11 @@ def resources_for_lesson_plan(verified_teacher)
     grouped_resources
   end
 
+  def lesson_plan_has_verified_resources
+    grouped_resources = resources.map(&:summarize_for_lesson_plan).group_by {|r| r[:audience]}
```

Comment:
nit: if you don't need to group the resources could you do something like `resources.any? {|r| r.audience == 'Verified Teacher'}`?

Reply:
Done!